content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -116,7 +116,7 @@ Operator.
 * [Charmed Operator Framework](https://juju.is/)
 * [kubebuilder](https://book.kubebuilder.io/)
 * [KUDO](https://kudo.dev/) (Kubernetes Universal Declarative Operator)
-* [Metacontroller](https://metacontroller.app/) along with WebHooks that
+* [Metacontroller](https://metacontroller.github.io/metacontroller/intro.html) along with WebHooks that
```

Comment:
The extra "/intro.html" is risky because we don't know how frequent the site is updated.